apps/api/src/hyp3_api/handlers.py

```diff
@@ -22,13 +21,7 @@ def problem_format(status, message):
     return response
 
 
-@draft4_format_checker.checks('uuid')
-def is_uuid(val):
-    try:
-        UUID(val, version=4)
-    except ValueError:
-        return False
-    return True
+is_uuid = draft4_format_checker.checks('uuid')(is_uuid)
```

Comment:
I had to move the `is_uuid`  to validation to avoid a circular import since we *need* to provide the `granules` parameter in every job spec, and I need a way to validate that a HyP3 job ID "granule" is a valid third part granule so it's okay CMR doesn't return metadata for the granule.

This should leave is_uuid in this module the same as the decorated function version.

Is there any other gotchas with `granules` we need to look out for?